src/tv2-common/actions/executeAction.ts

```diff
@@ -1955,6 +1958,58 @@ async function executeActionClearGraphics<
 	})
 }
 
+async function executeActionClearTemaGraphics(context: ActionExecutionContext) {
+	await context.core.stopPiecesOnLayers([SharedSourceLayer.PgmGraphicsTema])
+	await context.core.insertPiece('current', {
+		enable: {
+			start: 'now',
+			duration: 3000
+		},
+		externalId: 'clearTemaGFX',
+		name: 'GFX Temaud',
+		sourceLayerId: SharedSourceLayer.PgmAdlibGraphicCmd,
+		outputLayerId: SharedOutputLayer.SEC,
+		lifespan: PieceLifespan.WithinPart,
+		content:
+			context.config.studio.GraphicsType === 'HTML'
+				? {
+						timelineObjects: [
+							literal<TSR.TimelineObjAbstractAny>({
+								id: '',
+								enable: {
+									start: 0
+								},
+								priority: 1,
+								layer: SharedGraphicLLayer.GraphicLLayerAdLibs,
+								content: {
+									deviceType: TSR.DeviceType.ABSTRACT
+								}
+							})
+						]
+				  }
+				: {
+						timelineObjects: [
+							literal<TSR.TimelineObjVIZMSEElementInternal>({
+								id: '',
+								enable: {
+									start: 0
+								},
+								priority: 100,
+								layer: SharedGraphicLLayer.GraphicLLayerAdLibs,
+								content: {
+									deviceType: TSR.DeviceType.VIZMSE,
+									type: TSR.TimelineContentTypeVizMSE.ELEMENT_INTERNAL,
+									templateName: 'OUT_TEMA_H',
+									templateData: [],
+									showName: context.config.selectedGfxSetup.OvlShowName ?? ''
+								}
+							})
+						]
```

Comment:
I think this part is indentet one time to many. 
In general anything above 3 idents would indicate that something should be changed, atleast in my mind.
The changes that Lindved suggest just above sounds like something that would help with the indentation.

For fun, try to google 'Never Nester' :)